modules/cli/src/main/scala/scala/cli/commands/InstallCompletions.scala

```diff
@@ -17,7 +18,7 @@ object InstallCompletions extends ScalaCommand[InstallCompletionsOptions] {
     List("install", "completions"),
     List("install-completions")
   )
-  private lazy val home = os.Path(sys.props("user.home"), os.pwd)
+  private lazy val home = os.Path(sys.props("user.home"), os.home)
```

Comment:
I meant, there's no need to define this, `home` is just `os.home`.
```suggestion
  private lazy val home = os.Path(sys.props("user.home"), os.home)
```